libs/platform/src/lib/components/action-button-group/action-button-group.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { ActionButtonGroupComponent } from './action-button-group.component';
+import { Component, ViewChild, ElementRef } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { FundamentalNgxCoreModule } from '@fundamental-ngx/core';
+@Component({
+    selector: 'fdp-action-button-group-component',
+    template: `
+        <fdp-action-button-group  #actionbuttongroup>
+        <button>
+            Save
+        </button>
+        <button>
+            Cancel
+        </button>
+        </fdp-action-button-group>
+    `
+})
+
+class ActionButtonGroupTestComponent {
+    @ViewChild('actionbuttongroup', { static: false }) actionbuttongroup: ActionButtonGroupComponent;

```

Comment:
We dont need anymore to add `{static: false}` it's default value since new ng version

Reply:
removed.